File: online-judge-solutions/Codeforces/386A.cpp
//https://codeforces.com/problemset/problem/386/A
//Lang		:	GNU C++14
//Time		:	15 ms
//Memory	:	200 KB
#include<iostream>
#include<map>
using namespace std;
int main(){
	int n,p;
	while(cin>>n){
		map<int,int> m;
		for(int i=0;i<n;i++){cin>>p;m[p]=i;}
		cout<<(m.rbegin()->second)+1<<" "<<next(m.begin(),n-2)->first<<endl;
	}
	return 0;
}

File: online-judge-solutions/Codeforces/386A.py
#https://codeforces.com/problemset/problem/386/A
#Lang		:	Python 3.8
#Time		:	46 ms
#Memory         :	0 KB
n,m=int(input()),{}
p=list(map(int,input().split()))[:n]
m={x:i for i,x in enumerate(p)}
m=dict(sorted(m.items()))
print(m[list(m)[-1]]+1,list(m)[n-2])
